test/blackbox/pushpull_authn.bats

```diff
@@ -0,0 +1,145 @@
+load helpers_cloud
+
+function setup() {
```

Comment:
```suggestion
function setup_file() {
```
just to make sure login is used in all tests?

Reply:
Actually this is starting a new zot server for every test. Is this intended? If not, I think Sebi is right.
And the change should also be done in the `teardown()` -> `teardown_file()`